pkg/operator/observe_config.go

```diff
@@ -115,13 +168,36 @@ func (c ConfigObserver) sync() error {
 	return nil
 }
 
+func observeInternalRegistryHostname(listers Listers, observedConfig map[string]interface{}) (map[string]interface{}, error) {
+	configImage, err := listers.imageConfigLister.Get("cluster")
+	if errors.IsNotFound(err) {
+		return observedConfig, nil
+	}
+	if err != nil {
+		return nil, err
```

Comment:
@deads2k 
I have switched this pull to use listers, but even then, the apiserver operator is still failing to `GET` the config image here. Logs from my local apiserver-operator pod:

```
I1106 16:07:24.876715       1 reflector.go:240] Listing and watching *v1.Image from github.com/openshift/cluster-openshift-apiserver-operator/vendor/github.com/openshift/client-go/config/informers/externalversions/factory.go:101
I1106 16:07:24.876882       1 request.go:641] Error in request: v1.ListOptions is not suitable for converting to "config.openshift.io/v1"
```